modules/batch-tool/src/main/java/com/devonfw/module/batch/common/cli/ExecutionsCommand.java

```diff
@@ -0,0 +1,128 @@
+    return -1;
+  }
+
+  @Component
+  @Command(name = "status", description = "Set the status of the given execution.")
+  static class StatusCommand implements Callable<Integer> {
+
+    @Parameters(index = "0", paramLabel = "executionId", description = "ID of the execution to modify.")
+    private Long executionId;
+
+    @Parameters(index = "1", paramLabel = "status", description = "The new status you wish the execution to set to.")
+    private BatchStatus status;
+
+    @Inject
+    private JobExplorer jobExplorer;
+
+    @Inject
+    private JobRepository jobRepository;
+
+    @Override
+    public Integer call() throws Exception {
+
+      JobExecution jobExecution = this.jobExplorer.getJobExecution(this.executionId);
+      if (jobExecution == null) {
+        System.out.println(
+            "No execution with ID " + this.executionId + " found. Use list command to find existing executions.");
+        return -1;
+      }
+      jobExecution.setStatus(this.status);
+      this.jobRepository.update(jobExecution);
+      System.out.println("Done.");
+      return 0;
+    }
+
+  }
+
+  @Component
+  @Command(name = "list", description = "List all executions for the job.")
+  static class ListCommand implements Callable<Integer> {
+    @Parameters(index = "0", paramLabel = "jobName", description = "The name of the job, which executions should managed.")
+    private String jobName;
+
+    @Option(names = "--printParams", description = "Print the parameters of the execution.")
+    private boolean printParams;
+
+    @Option(names = "--printExit", description = "Print exit status of the execution.")
+    private boolean printExit;
+
+    @Inject
+    private JobExplorer jobExplorer;
+
+    @Override
+    public Integer call() throws NoSuchJobException {
+
+      System.out.print("ID\tStart Time\tEnd Time\tStatus");
+      if (this.printParams) {
+        System.out.print("\tParameters");
+      }
+      if (this.printExit) {
+        System.out.print("\tExit status");
+      }
+      System.out.println();
+      List<JobExecution> jobExecutions = new LinkedList<>();
+      for (JobInstance jobInstance : this.jobExplorer.findJobInstancesByJobName(this.jobName, 0,
+          this.jobExplorer.getJobInstanceCount(this.jobName))) {
+        jobExecutions.addAll(this.jobExplorer.getJobExecutions(jobInstance));
+      }
+
+      jobExecutions.sort((JobExecution e1, JobExecution e2) -> e1.getStartTime().compareTo(e2.getStartTime()));
+      for (JobExecution jobExecution : jobExecutions) {
+        System.out.print(jobExecution.getId());
+        System.out.print("\t" + jobExecution.getStartTime());
+        System.out.print("\t" + jobExecution.getEndTime());
+        System.out.print("\t" + jobExecution.getStatus());
+        if (this.printParams) {
+          System.out.print("\t" + jobExecution.getJobParameters());
+        }
+        if (this.printExit) {
+          System.out.print("\t" + jobExecution.getExitStatus());
+        }
```

Comment:
Why does this have to be activated explicitly? IMHO it does not hurt to always print out all the details.

Reply:
this code get very long and clutter the output

Reply:
the exit status is IMHO important and just one short line of output. I agree that job parameteres could be messy and here an option could help.

Reply:
Okay maybe that is a missunderstanding. The current default output is e.g.:
▶ java -D'spring.datasource.url=jdbc:h2:~/mts;AUTO_SERVER=TRUE' -jar devonfw-batch-tool.jar executions list countJob
ID	Start Time	End Time	Status
18	2020-01-21 14:20:04.425	2020-01-21 14:20:39.437	ABANDONED
...
35	2020-01-21 16:54:12.863	2020-01-21 16:54:17.37	COMPLETED
36	2020-01-29 14:00:12.714	2020-01-29 14:00:17.205	FAILED

ExitStatus is complex object in Spring Batch witch includes e.g. the stacktrace of the exception wich causes the batch to fail.

Reply:
OK. Indeed a mistake on my end. Cosidered done.

Reply:
Thanks for clarification and sorry for confusion.